netaddr.go

```diff
@@ -288,6 +289,42 @@ func (ip IP) Zone() string {
 	return ""
 }
 
+// Compare returns an integer comparing two IPs. The result will be 0 if ip==ip2, -1 if ip < ip2, and +1 if ip > ip2.
+// The definition of "less than" is the same as the IP.Less method.
```

Comment:
Any reason not to replace the implementation of Less with `return ip.Compare(ip2) == -1`?

Reply:
done